resources/scenarios/commander.py

```diff
@@ -135,9 +135,13 @@ def parse_args(self):
         help_parser = argparse.ArgumentParser(usage="%(prog)s [options]")
         self.add_options(help_parser)
         help_args, _ = help_parser.parse_known_args()
-        if help_args.help:
-            help_parser.print_help()
-            sys.exit(0)
+        try:
+            # This fails inside of CI, and I don't know why particularly
+            if help_args.help:
+                help_parser.print_help()
+                sys.exit(0)
+        except Exception:
+            pass
```

Comment:
this seems like a hack, would be better if we can figure out why CI is failing. happy to help troubleshoot this.

Reply:
This was fixed in https://github.com/bitcoin-dev-project/warnet/pull/552 after the failure was diagnosed fully.

I will rebase this shortly